mxnet-bot/PredictLabels/application.py

```diff
@@ -57,7 +57,9 @@ def predict():
     # data would be a json file {"issues":[1,2,3]}
     data = request.get_json()["issues"]
     # predictions = predict_labels.predict(data)
-    predictions = predictor.predict(data)
+    predictions = []
+    if len(data) != 0:
```

Comment:
is data a list?  Could we just do
```python
if data:
```

Reply:
Yes, the data is a list. I am not familiar with Python. Does `if data` means if data is `not null` or `data isEmpty`?